### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by cheapwebmonkey**
The Open Source Geospatial Foundation (OSGeo) is a not-for-profit organization whose mission is to foster global adoption of open geospatial technology by being an inclusive software foundation devoted to an open philosophy and participatory community driven development.

### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [x] Suggesting edits to an existing topic or collection
  - [ ] Curating a new topic or collection

---------------------------------------------------------------------

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288)
- [x] Content (and my changes are in `index.md`)

> This change adds OSGeo on Github to the collection.

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> This change helps bring more visibility and awareness to OSGeo, invaluable resource for open geospatial technology, in this community.

---------------------------------------------------------------------

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: collections/open-source-organizations/index.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ items:
  - IBM/ibm.github.io
  - Microsoft/microsoft.github.io
  - artsy/artsy.github.io
+ - OSGeo/github.com/OSGeo
```
Comment (MikeMcQuaid) [inline collections/open-source-organizations/index.md]: Can this be updated to OSGeo/OSGeo? Thanks!

Comment (cheapwebmonkey) [reply collections/open-source-organizations/index.md]: Updated :)

**Comment by ramlaxman** [comment]
Hi @cheapwebmonkey,
As per my observation @MikeMcQuaid is suggesting to add this line:

OSGeo/osgeo

Just add above line in place of  

OSGeo/OSGeo/OSGeo

and you are good to go.

And BTW, welcome to Github Explore !

File: collections/open-source-organizations/index.md [hunk 2]
```diff
@@ -14,6 +14,7 @@ items:
  - IBM/ibm.github.io
  - Microsoft/microsoft.github.io
  - artsy/artsy.github.io
+ - OSGeo/OSGeo/OSGeo
```
Comment (MikeMcQuaid) [inline collections/open-source-organizations/index.md]: Sorry! This needs to be `OSGeo/OSGeo` which is the user or organisation, a `/` and then the repository (like those above)

**Comment by cheapwebmonkey** [comment]
adjusted

**Comment by MikeMcQuaid** [comment]
Thanks so much for your first contribution! Without people like you submitting PRs we couldn't run this project. You rock, @cheapwebmonkey!

**Pull request merged**